src/sagas/initialize.js

```diff
@@ -91,6 +92,14 @@ export function* handleInitSuccess(action, path = '/form/identification/intro')
     yield put(updateApplication('Settings', 'status', Status))
     yield put(updateApplication('Settings', 'hash', Hash))
 
+    const { headers } = response
+    const csrfToken = headers['x-csrf-token']
```

Comment:
I haven't dived deeply here before – what mechanism ensures the headers are all lower case at this point?

Reply:
It's a property of Axios, the promise http requests library we are using.